sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -868,11 +900,11 @@ public void onCallSucceeded(final String payload) {
 
                     @Override
                     public void run() {
-                        try {
-                            handleApiCallSuccess(releaseCallId, payload, ReleaseDetails.parse(payload));
-                        } catch (JSONException e) {
-                            onCallFailed(e);
-                        }
+                    try {
+                        handleApiCallSuccess(releaseCallId, payload, ReleaseDetails.parse(payload));
+                    } catch (JSONException e) {
+                        onCallFailed(e);
+                    }
```

Comment:
This was appearing as double-tabbed in my editor...?

Reply:
Not sure what you mean. If it's a formatting issue, feel free to reformat using Android Studio and commit.